main_window.py

```diff
@@ -357,7 +357,7 @@ def create_downloading_firmware_window(self):
         self.downloading_window_status = 1
         self.downloading_window.overrideredirect(True)
 
-        self._main_window.attributes('-disable', True)
```

Comment:
Unknown Tinker option.  Supported options are as follows:

_tkinter.TclError: bad attribute "-disable": must be -alpha, -fullscreen, -modified, -notify, -titlepath, -topmost, -transparent, or -type